### PR TITLE
Check for admin or manage server permissions

### PR DESCRIPTION
**Pull request by Derekholio**

### CODE REVIEW

File: src/SrBot.ts [hunk 1]
```diff
@@ -222,12 +222,12 @@ export class SrBot {
...
      * @param user Username to check
      */
-    private isServerAdmin(serverId: string, user: string): boolean {
+    private isServerAdmin(serverId: string, user: string, permission: Discord.Permissions): boolean {
         const server = this.statsGenerator.getServer(serverId);
         if (!server){
             return false;
         }
-        return server.admins.map(user => user.toUpperCase()).includes(user.toUpperCase());
+        return server.admins.map(user => user.toUpperCase()).includes(user.toUpperCase()) || permission.has('ADMINISTRATOR') || permission.has('MANAGE_GUILD');
```
Comment (rypete) [inline src/SrBot.ts]: Do you want to remove `.includes(user.toUpperCase()` in case of username switching?  If so, remove it from overwatch.json as well

Comment (rypete) [reply src/SrBot.ts]: Also what is the is the `permission.has()` based on?  the sender?

Comment (Derekholio) [reply src/SrBot.ts]: Good catch. I meant to switch those to IDs. And the permissions are based on the sender. It is looking for these built in discord permissions:
image

Comment (rypete) [reply src/SrBot.ts]: Should we just drop the id/username check altogether then?  I think channel perms would be the more solid approach.

`isAdmin(permissions: ...)`

Comment (Derekholio) [reply src/SrBot.ts]: Yeah, I think so. We can always revisit it later if needed

**Review by rypete — APPROVED** [review]

**Pull request merged**